linux/README.md

```diff
@@ -0,0 +1,107 @@
+# The Flutter Linux Desktop Embedder
+
+This framework provides the basic functionality for embedding a Flutter app on
+the Linux desktop. This currently includes support for:
+
+*  Drawing a Flutter view.
+*  Mouse event handling.
+
+# How to Use This Code
+
+Note that the example build here is `host_debug_unopt` (you'll find more info on
+what that means in this section). This can be changed to whatever version of
+the Flutter Engine you end up building.
+
+First you will need to install the relevant dependencies.
+
+## Dependencies
+
+Requires GLFW3 on your system (example for debian-based systems):
+
+```
+$ sudo apt-get install libglfw3-dev
+```
+
+Next you will need to follow the steps outlined on the flutter engine's
+[contributing](https://github.com/flutter/engine/blob/master/CONTRIBUTING.md)
+document so that you can build a copy of `libflutter_engine.so`.
+
+Then copy it in the `linux/library/` directory.
```

Comment:
Seems like we should have a .gitignore checked in that will ignore this and the build output.

I'd also like to revisit this later, and consider putting it outside the checkout the way we do for the fetched macOS framework, since it's nice to not force people to keep persistent state mixed with a checkout. But we can talk about that next week and do a follow-up CL if we go that route.

Reply:
Yeah, it seems like a bit of a pain to have people keep these things around in this directory (and it's a bit convoluted).  I've added a .gitignore for now, though